test/rest/json2inputs.pl6

```diff
@@ -0,0 +1,30 @@
+use v6;
+
+use JSON::Fast;
+
+sub MAIN(Str $filename) {
+
+    my %input-data = from-json($filename.IO.slurp)<inputs>;
+    for %input-data.kv -> $full-tax, $module-data {
+        note $full-tax;
+        if $full-tax.index('[') -> $sub-start {
```

Comment:
This also seems to use the old JSON format.

Reply:
Deleted (together with some other test scripts and test data files.